src/api/myanimelist.ts

```diff
@@ -0,0 +1,45 @@
+import * as request from "request";
+import rp from "request-promise";
+import { StringNullableChain } from "lodash";
```

Comment:
almost all javascript devs generally import _ from 'lodash', its so common that its better to do it to make other devs comfortable with the code.

You're also importing request-promise but not using it, you're essentially doing your own version of request promise by wrapping the callback version of request in your own Promise. I suggest just using rp here, you don't need to import regular request, its a peer dependency but you don't need to import it